### DISCUSSION TITLE
Tree控件展开怎么获取父节点

### DISCUSSION THREAD

**Discussion by snowrain0722** [ant-design/ant-design#43797] [Q&A]
主要做文件夹目录需求，展开的节点要放到本地存储中，主要的卡点问题是怎么在Tree组件的onExpand获取父节点key，key是服务端获取的纯数字ID。

**Comment by linxianxi** [comment]
类似的道理  https://ant.design/components/tree-select-cn#onchange-%E6%97%B6%E5%A6%82%E4%BD%95%E8%8E%B7%E5%BE%97%E7%88%B6%E8%8A%82%E7%82%B9%E4%BF%A1%E6%81%AF
